tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def pytest_addoption(parser):
```

Comment:
Simple question on the use of `addopts` and `addoption`. Is this a common wording practice? I only ask because it took me a second to understand the variable name since I was catch up with seeing this variable as `adopt` and `adoption`.

Reply:
I am not entirely sure to be honest.  I got this after researching how to add options to pytest so that I could pass in different config files for different platforms.  Looking around now, it does appear that exact way in many different places that show how to add options to pytest.  I agree with you that the naming is not the best, but I think I'm going to leave it since every where I look, it's done that way.